scintillometry/utils/phase_utils.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import warnings
+
+import numpy as np
+import astropy.units as u
+import astropy.time as time
+
+try:
+    from .pint_utils import PintUtils
+except ModuleNotFoundError:
```

Comment:
I think this should be `except ImportError` to get rid of the failure.

Alternatively, and perhaps better, you could do the import inside `PintPhase.__init__`, so that it doesn't even try unless the user actually wants to use PINT. In that case, of course you do not need to check for the import error at all.